src/Users/UserStatistics.php

```diff
@@ -0,0 +1,341 @@
+<?php
+
+namespace Scriptotek\Alma\Users;
+
+use Scriptotek\Alma\Client;
+use Scriptotek\Alma\Model\Model;
+
+/**
+ * Iterable collection of statistics resources.
+ */
+class UserStatistics extends Model
+{
+    /**
+     * Get the user's statistics.
+     *
+     * @return Array of Statistics.
+     */
+    public function getStatistics()
+    {
+        $stats = array();
+        foreach ($this->data->user_statistic as $statistic) {
+            $stats[] = $statistic;
+        }
+        return $stats;
+    }
+
+    /**
+     * Get the user's statistics.
+     * 
+     * @return array of statistics.
+     */
+    public function get()
+    {
+        $stats = array();
+        foreach ($this->data->user_statistic as $statistic) {
+            $stats[] = $statistic;
+        }
+        return $stats;
+    }
+
+    /**
+    * Get a statistic.
+    *
+    * @param $typeCode code of the category type.
+    * @param $categoryCode code of the statistical category.
+    *
+    * @return Statistic.
+    */
+    public function getStatistic($typeCode,$categoryCode)
```

Comment:
Confirm whether multiple Statistics can really exist across a single type/category combination.

Also, compare with similar method signatures in codetables, identifiers, etc.  In codetables, for example, there is a pattern of:
```
// get a single entry
$client->codetable->get( $code );
// get all:
$client->codetable->getCodeTables( );
```
Is this typical?  If so, should this look like:
```
// get one
$user->Statistics->get( $type, $category );
// get all
$user->Statistics->getAll( );
```